apps/src/templates/manageStudents/PasswordReset.jsx

```diff
@@ -0,0 +1,78 @@
+import React, {Component, PropTypes} from 'react';
+import Button from '../Button';
+import i18n from "@cdo/locale";
+
+const styles = {
+  input: {
+    width: 100,
+    height: 29,
+    marginTop: '-25px',
+    marginRight: 10,
+  }
+};
+
+class PasswordReset extends Component {
+  static propTypes = {
+    resetAction: PropTypes.func.isRequired,
+    isResetting: PropTypes.bool
```

Comment:
Brad has recommended to me before that when using a prop as an initial state we name it differently to avoid confusion, i.e. something like `initialIsResetting` or `defaultIsResetting`